libs/applications/data-access/src/lib/services/grpc-applications.service.ts

```diff
@@ -54,10 +56,7 @@ export class GrpcApplicationsService extends BaseGrpcService {
     return {
       page: page !== 0 ? page : undefined,
       pageSize: pageSize !== 10 ? pageSize : undefined,
-      orderBy:
-        orderBy !== ListApplicationsRequest.OrderByField.ORDER_BY_FIELD_NAME
-          ? orderBy
-          : undefined,
+      orderBy,
```

Comment:
May you take a look at this solution ? I find it more pleasant to have a minimal number of query params.
```suggestion
      orderBy:
        orderBy.length === 1 &&
        !orderBy.includes(ListApplicationsRequest.OrderByField.ORDER_BY_FIELD_NAME)
          ? orderBy
          : undefined,
```